Alpine-specific patch; Look for cgdb.txt where it's packaged.

--- a/cgdb/interface.cpp
+++ b/cgdb/interface.cpp
@@ -1566,7 +1566,7 @@
     char cgdb_help_file[FSUTIL_PATH_MAX];
     int ret_val = 0;
 
-    fs_util_get_path(PKGDATADIR, "cgdb.txt", cgdb_help_file);
+    fs_util_get_path("/usr/share/doc/cgdb", "cgdb.txt", cgdb_help_file);
 
     /* File doesn't exist. Try to find cgdb.txt in the build dir in case
      * the user is running a built cgdb binary directly. */
